src/routes/main.routes.ts

```diff
@@ -15,4 +15,4 @@ class MainRoutes {
   }
 }
 
-export const mainRoutes: express.Router = new MainRoutes().router;
\ No newline at end of file
+export const mainRoutes: express.Router = new MainRoutes().router;
```

Comment:
Change to normal function.
```
export const mainRoutes = () => {
   // todo: create routes and return them
}
```

Reply:
I've been seeing this patter in TypeScript projects where almost everything is a class. Can you please advice on which is the most suitable instance of using a class and why this one is not? @biblicalph